uber/model_checks.py

```diff
@@ -197,14 +249,17 @@ def job_conflicts(job):
                 return 'You cannot change this job to this time, because {} is already working a shift then'.format(shift.attendee.full_name)
 
 
-cashformpoints_amount = money_amount
-
+@validation.OldMPointExchange
 def oldmpointexchange_numbers(mpe):
     if not str(mpe.amount).isdigit():
         return 'MPoints must be a positive integer'
 
-sale_required = [('what',"What's being sold")]
-def sale_amounts(sale):
+
+Sale.required = [('what', "What's being sold")]
+
+
+@validation.Sale
+def cash_and_mpoints(sale):
     if not str(sale.cash).isdigit() or int(sale.cash) < 0:
         return 'Cash must be a positive integer'
     if not str(sale.mpoints).isdigit() or int(sale.mpoints) < 0:
```

Comment:
Can you elaborate on switching the directionality of `relationship` declarations?